app/src/main/java/io/apicurio/registry/content/ContentCanonicalizerFactory.java

```diff
@@ -60,8 +64,12 @@ public ContentCanonicalizer create(ArtifactType type) {
                 return noop;
             case GRAPHQL:
                 return graphql;
-            default:
-                break;
+            case XSD:
+                return xsd;
+            case WSDL:
+                return wsdl;
+            case XML:
+             return xml;
```

Comment:
Good - just fix the indentation of line 72 - the `return xml;` is not properly indented.